packages/runtime/container-runtime/src/test/summarizerNode.spec.ts

```diff
@@ -439,35 +439,10 @@ describe("Runtime", () => {
 						readAndParseBlob,
```

Comment:
It's weird that this code passes in params to `refreshLatestSummary` which have been removed.